components/molecules/FilterCardSelect/filter-card-select.tsx

```diff
@@ -4,10 +4,11 @@ import Image from "next/image";
 import { BsFillCheckCircleFill } from "react-icons/bs";
 
 import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "components/atoms/Select/select";
-import hashIcon from "../../../img/icons/hash.svg";
+// import hashIcon from "../../../img/icons/hash.svg";
```

Comment:
dead code needs to be removed.

Reply:
@bdougie Please checkout the latest commit . 
i have remove the comment code and fixed the hash.svg ....
Earlier i was using a different svg code from ....but now i have fixed it with the same code as it was before